### PR TITLE
Add Jenkins ID to submission table

### PR DESCRIPTION
**Pull request by mike-ferguson**
Adds a `jenkins_id` column to submission table in DB, auto-increments the primary key (`ID`)
      
Some things to note:

1. The reason for this change is that language models now have their own Jenkins jobs that start at 0. If no changes were made, then the submission IDs (Jenkins job #s) for language would start to override the vision entries.
2. As of 3/9/2023, all submissions 6756 and below will have their old primary key (the previous` ID `column that was generated automatically by Django) duplicated in the new `jenkins_id` column. This preserves old `ID`s for models before language integration (models 6756 and below).
3. `ID` column now will start auto-incrementing at 6756
4. `ID` and `jenkins_id` column values will be identical up until 6756 ID. after that, they will diverge due to language models being scored.
5. The purpose of the `jenkins_id` column is simply to preserve that information in case we need it

### CODE REVIEW

File: benchmarks/models.py [hunk 1]
```diff
@@ -155,11 +155,16 @@ class Status:
         SUCCESS = 'successful'
         FAILURE = 'failure'
 
+    # ID now auto-increments
+    id = models.AutoField(primary_key=True)
```
Comment (mschrimpf) [inline benchmarks/models.py:159]: I believe we can just leave this out, django will automatically create
image

File: benchmarks/models.py [hunk 2]
```diff
@@ -155,11 +155,16 @@ class Status:
...
     submitter = models.ForeignKey(User, on_delete=models.PROTECT)
     timestamp = models.DateTimeField(auto_now_add=True, blank=True)
     model_type = models.CharField(max_length=30, default='BaseModel')
     status = models.CharField(max_length=25, default='unknown')
 
+    # equivalent to ID until language changes were added: (ID 6756)
+    jenkins_id = models.IntegerField(null=True)
```
Comment (mschrimpf) [inline benchmarks/models.py:166]: why can it be `null`?

File: benchmarks/models.py [hunk 3]
```diff
@@ -155,11 +155,16 @@ class Status:
...
         FAILURE = 'failure'
 
+    # ID now auto-increments
+    id = models.AutoField(primary_key=True)
     submitter = models.ForeignKey(User, on_delete=models.PROTECT)
     timestamp = models.DateTimeField(auto_now_add=True, blank=True)
     model_type = models.CharField(max_length=30, default='BaseModel')
     status = models.CharField(max_length=25, default='unknown')
 
+    # equivalent to ID until language changes were added: (ID 6756)
```
Comment (mschrimpf) [inline benchmarks/models.py:165]:
```suggestion
    # until https://github.com/brain-score/brain-score.web/pull/159, this used to be the `id` field (last id before change: 6756)
```

File: benchmarks/migrations/0007_auto_20230309_1047.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+        migrations.AddField(
+            model_name='submission',
+            name='jenkins_id',
+            field=models.IntegerField(null=True),
+        ),
+        migrations.AlterField(
+            model_name='submission',
+            name='id',
+            field=models.AutoField(primary_key=True, serialize=False),
+        ),
```
Comment (mschrimpf) [inline benchmarks/migrations/0007_auto_20230309_1047.py:22]: shouldn't it alter `id` -> `jenkins_id` and add a new `id` for primary key? Possible it doesn't work this way; if it doesn't we'll need to move all `id` values into `jenkins_id`

Comment (mike-ferguson) [reply benchmarks/migrations/0007_auto_20230309_1047.py:22]: After trying, it is indeed odd to do it the first way, so I will do it the second way you mentioned. It is easier and faster

File: benchmarks/migrations/0007_submission_jenkins_id.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('benchmarks', '0006_model_competition'),
+    ]
+
+    operations = [
+        migrations.AddField(
```
Comment (mschrimpf) [inline benchmarks/migrations/0007_submission_jenkins_id.py:13]: should this not be `AlterField` to rename the current `id`?

File: benchmarks/models.py [hunk 6]
```diff
@@ -160,6 +160,9 @@ class Status:
...
+    jenkins_id = models.IntegerField(default=999)
```
Comment (mschrimpf) [inline benchmarks/models.py:164]:
```suggestion
    jenkins_id = models.IntegerField()
```
don't see why this needs a default?

**Pull request merged**